source/mainnet/smart-contracts/tutorials/piggy-bank/deploying.rst

```diff
@@ -192,10 +192,12 @@ If everything works correctly the output has a green line as follows:
 
 You can find additional information about updating a smart contract instance in this :ref:`guide<interact-instance>`.
 
+``Concordium-std`` crate errors
+-------------------------------
+
 .. note::
```

Comment:
I don't think the first paragraph in each of these error sections should be notes. I find it odd to have a section that is mostly side-notes.
So just remove the `.. note::` line for the first paragraph in the two error sections 😊